.github/workflows/gradle-android-tests.yml

```diff
@@ -1,84 +1,86 @@
+              uses: reactivecircus/android-emulator-runner@v2
+              with:
+                  api-level: ${{ matrix.api-level }}
+                  force-avd-creation: false
+                  target: google_apis
+                  emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
+                  disable-animations: false
+                  script: echo "Generated AVD snapshot for caching."
 
-      - name: Android Instrumentation Tests
-        uses: reactivecircus/android-emulator-runner@v2
-        with:
-          api-level: ${{ matrix.api-level }}
-          script: ./gradlew connectedAndroidOnlyAffectedTest
-        env:
-            GITHUB_USER: ${{ github.actor }}
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            - name: Android Instrumentation Tests
+              uses: reactivecircus/android-emulator-runner@v2
+              with:
+                  api-level: ${{ matrix.api-level }}
+                  target: google_apis
+                  script: ./gradlew connectedAndroidOnlyAffectedTest
+              env:
+                  GITHUB_USER: ${{ github.actor }}
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Archive Test Reports
-        if: always()
-        uses: actions/upload-artifact@v3
-        with:
-            name: test-reports
-            path: ./**/build/reports/tests/**
+            - name: Archive Test Reports
+              if: always()
+              uses: actions/upload-artifact@v3
+              with:
+                  name: test-reports
+                  path: ./**/build/reports/tests/**
 
-      - name: Publish Unit Test Results
-        uses: EnricoMi/publish-unit-test-result-action/composite@v1.25
-        if: always()
-        with:
-          files: |
-            **/build/test-results/**/*.xml
-            **/build/outputs/androidTest-results/**/*.xml
+            - name: Publish Unit Test Results
+              uses: EnricoMi/publish-unit-test-result-action/composite@v1.25
+              if: always()
+              with:
+                  files: |
+                      **/build/test-results/**/*.xml
+                      **/build/outputs/androidTest-results/**/*.xml
 
-      - name: Cleanup Gradle Cache
-        # Remove some files from the Gradle cache, so they aren't cached by GitHub Actions.
-        # Restoring these files from a GitHub Actions cache might cause problems for future builds.
-        run: |
-          rm -f ~/.gradle/caches/modules-2/modules-2.lock
-          rm -f ~/.gradle/caches/modules-2/gc.properties
+            - name: Cleanup Gradle Cache
+                # Remove some files from the Gradle cache, so they aren't cached by GitHub Actions.
+                # Restoring these files from a GitHub Actions cache might cause problems for future builds.
+              run: |
+                  rm -f ~/.gradle/caches/modules-2/modules-2.lock
```

Comment:
Does the IDE like this 4 space indentation for YML files?

2 spaces is the standard for YML files and all of our other files in the project are 2 spaces as well.

Kinda annoying when moving code between files if the indentation doesn't match.

Reply:
@vitorhugods not sure what is happening I copy the content of this file from the develop changes it in 3 places and push it and this happens ;s